Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/security-emailthreatsubmissionpolicy-post-emailthreatsubmissionpolicies.md

---
title: "Create emailThreatSubmissionPolicy"
description: "Create a new emailThreatSubmissionPolicy object."
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

# Create emailThreatSubmissionPolicy
Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Create a new [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object.

## Permissions

> One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

Reviewer: ```suggestion
One of the following permissions is required to call this API. To learn more permissions, including how to choose one, see [Permissions](/graph/permissions-reference).
```

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|ThreatSubmissionPolicies.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported|
|Application|ThreatSubmissionPolicy.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
POST /security/threatSubmission/emailThreatSubmissionPolicies
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|
|Content-Type|application/json. Required.|

## Request body

> In the request body, supply a JSON representation of the [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object.

Reviewer: ```suggestion
In the request body, you need to provide a JSON representation of the [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object.
```

> You can specify the following properties when creating an **emailThreatSubmissionPolicy**.

Reviewer: ```suggestion
You may also specify the following properties when creating an **emailThreatSubmissionPolicy**.
```

> You may also specify the following properties when creating an **emailThreatSubmissionPolicy**.

Reviewer: ```suggestion
You may also specify the following properties when creating an **emailThreatSubmissionPolicy**. These properties are related with the user reported message settings and to know more check out this documentation - https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/user-submission?view=o365-worldwide
```

|Property|Type|Description|
|:---|:---|:---|

> | isReportToMicrosoftEnabled               | Boolean | If enabled, the email will be sent to microsoft for analysis.                              |

Reviewer: ```suggestion
| isReportToMicrosoftEnabled               | Boolean |It is used to specify whet                              |
```
```suggestion
| isReportToMicrosoftEnabled               | Boolean | It is used to specify whether the email messages reported by end users as junk, phish or not junk be sent to Microsoft or not?|
```

> | isReportToCustomizedEmailAddressEnabled  | Boolean | If enabled, the email will be sent to customizedReportRecipientEmailAddress.               |

Reviewer: ```suggestion
| isReportToCustomizedEmailAddressEnabled  | Boolean | It is used to specify whether emals reported by end users should be send to the custom mailbox which was configured using customizedReportRecipientEmailAddress? | 
```

> | isAskMeEnabledForUsers                   | Boolean | User could use 'Ask me before reporting the message' option. The default value is true.    |

Reviewer: ```suggestion
| isAskMeEnabledForUsers                   | Boolean | It is used to specify whether end users can confirm using a popup before reporting messages as spam, phish or not junk. Its default value is true |
```

> | isAlwaysReportEnabledForUsers            | Boolean | User could use 'Always report the message' option. The default value is true.              |

Reviewer: ```suggestion
| isAlwaysReportEnabledForUsers            | Boolean | It is used to specify whether end users can report a message as spam, phish or junk directly without a confirmation(popup). Its default value is true |
```

> | isNeverReportEnabledForUsers             | Boolean | User could use 'Never report the message' option. The default value is true.               |

Reviewer: ```suggestion
| isNeverReportEnabledForUsers             | Boolean | It is used to specify whether end users can submit from the quarantine page or not? |
```

> | isCustomizedMessageEnabledForPhishing    | Boolean | If enabled, customized message only shows when email is reported as phishing.              |

Reviewer: ```suggestion
| isCustomizedMessageEnabledForPhishing    | Boolean | It is used to specify whether the customized email notifications sent to end users is to be sent for phish or not? |
```

> | isCustomizedMessageEnabled               | Boolean | If enabled, customized message is enabled.                                                 |

Reviewer: ```suggestion
| isCustomizedMessageEnabled               | Boolean |It is used to specify whether the email notifications sent to end users to inform them whether an email is phish, spam or junk is customized or not? |
```

> | customizedReportRecipientEmailAddress    | String  | When customized address enabled, the message will be sent to this address.                 |

Reviewer: ```suggestion
| customizedReportRecipientEmailAddress    | String  | It is used to specify the destination where the reported messages from end users will land whenever they report something as phish, junk or not junk.   |
```

> | isReviewEmailNotificationEnabled         | Boolean | If enabled, system sends email notification to reporter when admin reviews the submission. |

Reviewer: ```suggestion
| isReviewEmailNotificationEnabled         | Boolean | If enabled, system sends email notification to reporter when admin reviews the submission. |
```
```suggestion
| isReviewEmailNotificationEnabled         | Boolean |It is used to specify whether an email notification be sent to the end user who reported the email when it has been reviewed by the Admin? |
```

> | isCustomizedNotificationSenderEnabled    | Boolean | If enabled, notificationSenderAddress is used by review notification email.                |

Reviewer: ```suggestion
| isCustomizedNotificationSenderEnabled    | Boolean | It is used to specify whether we should use the sender email address which we set earlier using customizedNotificationSenderEmailAddress for sending email notifications to end users or not? |
```

> | isOrganizationBrandingEnabled            | Boolean | If enabled, tenant branding logo will be used in review notification message.              |

Reviewer: ```suggestion
| isOrganizationBrandingEnabled            | Boolean | It is used to specify whether the branding logo should be used in the email notifications or not which are being sent to end users? |
```

> | customizedNotificationSenderEmailAddress | String  | Customized notification sender address for review notification message.                    |

Reviewer: ```suggestion
| customizedNotificationSenderEmailAddress | String  | It is used to specify the email address of the sender from which email notifications will be sent to end users to inform them whether an email is spam, phish or clean. |
```

> | isReportFromQuarantineEnabled            | Boolean | If enabled, submission could be created from security portal quarantine page.              |

Reviewer 2: Please order these properties alphabetically.

> | isReportFromQuarantineEnabled            | Boolean | If enabled, submission could be created from security portal quarantine page.              |

Reviewer: ```suggestion
| isReportFromQuarantineEnabled            | Boolean | It is used to specify whether end users can submit from the quarantine page or not? |
```

## Response

If successful, this method returns a `201 Created` response code and an [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) object in the response body.

## Examples

### Request
The following is an example of a request.
<!-- {
  "blockType": "request",
  "name": "create_emailthreatsubmissionpolicy_from_emailthreatsubmissionpolicies"
}
-->
``` http
POST https://graph.microsoft.com/beta/security/threatSubmission/emailthreatSubmissionPolicies
Content-type: application/json

{
  "isReportToMicrosoftEnabled": true
}

> ```

Reviewer 2: In the example request: the api.md file says that all the properties of the entity are required, but in this example, we are only initializing one. 

Are the other properties optional? Do they have default values that populate automatically? Can they be null? 

The answer to those questions should be answered in the properties table of the resource topic, in the description column. 

Please add a sentence to the description column to clarify whether there are default values to these properties, or if they support null or if they are optional.

---

> title: "Create emailThreatSubmissionPolicy"

Reviewer 3: I'm not sure that this file name is correct. It looks like emailThreatSubmissionPolicies is a nav property of threatSubmissionRoot, in which case we would have the file name security-emailthreatsubmissionroot-post-emailthreatsubmissionpolicies? I actually think the file name should just be security-emailthreatsubmissionpolicy-post.

Author: @Lauragra  the file name is generated **by the graph pipeline**.  The url path is /security/threatSubmission/emailThreatSubmissionPolices.
